src/modules/flow/form/form.c

```diff
@@ -3471,6 +3510,12 @@ string_formatted_select_set(struct sol_flow_node *node,
     return sol_flow_send_string_slice_packet(node,
         SOL_FLOW_NODE_TYPE_FORM_INT__OUT__SELECTED,
         sol_buffer_get_slice(&mdata->formatted_value));
+#else
+    sol_flow_send_error_packet(node, ENOTSUP, "The string-formatted node"
+        " won't work on this Soletta build -- Python-like formatting "
+        "capability was disabled");
+    return -EINVAL;
+#endif
```

Comment:
I don't like much having dependencies / configuration for only part of the node types of a module.

So you have the module, correct version, but it will give a runtime error for lack of support in some configuration. How should a flow deal with such error packet? Is there any solution?

Wouldn't make more sense to provide it in another module?